tools/tenscan/frontend/src/services/useTransactionsService.ts

```diff
@@ -96,3 +97,7 @@ export const useTransactionsService = () => {
     price,
   };
 };
+
+function jsonHexToObj(hex: string) {
+  return JSON.parse(Buffer.from(hex.slice(2), "hex").toString());
+}
```

Comment:
i guess this should be a util rather than kept in this file, so it can be reused, if needed.